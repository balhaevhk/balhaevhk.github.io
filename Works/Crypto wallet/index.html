<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Crypto Wallet</title>
    <link rel="stylesheet" href="css/main.css">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
</head>
<body>
    <div class="dashboard">
        <div class="dashboard__nav">
            <div class="nav">
                <div class="nav__logo">
                    <a href="#" class="logo">
                        <img src="./img/nav-icons/logo.svg" alt="logo">
                    </a>
                </div>
                <div class="nav__icons">
                    <ul class="nav__icons-list">
                        <li> 
                            <a href="#"><img src="./img/nav-icons/dashboard.svg" alt="dashboard"></a>
                        </li>
                        <li> 
                            <a href="#"><img src="./img/nav-icons/wallet.svg" alt="wallet"></a>
                        </li>
                        <li> 
                            <a href="#"><img src="./img/nav-icons/charts.svg" alt="charts"></a>
                        </li>
                        <li> 
                            <a href="#"><img src="./img/nav-icons/cube.svg" alt="cube"></a>
                        </li>
                        <li> 
                            <a href="#"><img src="./img/nav-icons/user.svg" alt="user"></a>
                        </li>
                        <li class="nav__icons-list-last-icon"> 
                            <a href="#"><img src="./img/nav-icons/login.svg" alt="login"></a>
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        <div class="dashboard__body">
            <div class="dashboard__header">
                <h1 class="title-1">Owerview</h1>
                <div class="dashboard__header-icons">
                    <button class="header-icon">
                        <img src="./img/top-icons/search.svg" alt="Search icon">
                    </button>
                    <button class="header-icon header-icon--new">
                        <img src="./img/top-icons/bell.svg" alt="Notifications icon">
                    </button>
                    <div class="user-profile">
                        <a href="#" class="user-profile__avatar">
                            <img src="./img/top-icons/avatar.svg" alt="Avatar">
                        </a>
                        <button class="user-profile__nav-button">Zoia M.</button>
                    </div>
                </div>
            </div>
            <div class="dashboard__cards">
                <div class="portfolio">
                    <div class="portfolio__header">
                        <h2 class="title-3">
                            Portfolio
                        </h2>
                    </div>
                    <div class="portfolio__card">
                        <div class="balance-card">
                            <div class="balance-card__balance">
                                <div class="balance-card__balance-value">$ 17 643.41</div>
                                <div class="balance-card__balance-desc">Portfolio balance</div>
                            </div>
                            <div class="balance-card__settings">
                                <button class="button-more">
                                    <span class="button-more__icon"></span>
                                </button>
                            </div>                      
                            <div class="balance-card__period">
                                <button class="balance-card__period-btn">1H</button>
                                <button class="balance-card__period-btn">24H</button>
                                <button class="balance-card__period-btn balance-card__period-btn--active">1W</button>
                                <button class="balance-card__period-btn">1M</button>
                                <button class="balance-card__period-btn">1Y</button>
                                <button class="balance-card__period-btn">ALL</button>
                            </div>
                            <img src="./img/chart.svg" alt="chart" class="balance-card__chart">
                        </div>
                    </div>
                </div>
                <div class="assets">
                    <div class="assets__header">
                        <h2 class="title-3">Your Assets</h2>
                        <button>
                            <img src="./img/top-icons/settings.svg" alt="settings">
                        </button>
                    </div>
                    <div class="assets__cards">
                        <div class="asset__card">
                            <div class="asset__card-header">
                                <div class="asset__card-value">1.25 BTC</div>
                                <div class="asset__card-fiat">$ 2948.04</div>
                            </div>
                            <div class="asset__card-settings">
                                <button class="button-more">
                                    <span class="button-more__icon"></span>
                                </button>
                           </div>
                            <div class="asset__card-footer">
                                <div class="asset__card-icon">
                                    <img src="./img/cards-icon/bitcoin.svg" alt="bitcoin">
                                </div>
                                <div class="asset__card-income">+ 0.14%</div>
                            </div>
                        </div>

                        <div class="asset__card asset__card--ltc">
                            <div class="asset__card-header">
                                <div class="asset__card-value">0.32 LTC</div>
                                <div class="asset__card-fiat">$ 2948.04</div>
                            </div>
                            <div class="asset__card-settings">
                                <button class="button-more">
                                    <span class="button-more__icon"></span>
                                </button>
                           </div>
                            <div class="asset__card-footer">
                                <div class="asset__card-icon">
                                    <img src="./img/cards-icon/litecoin.svg" alt="litecoin">
                                </div>
                                <div class="asset__card-income">+ 0.31%</div>
                            </div>
                        </div>

                        <div class="asset__card asset__card--eth">
                            <div class="asset__card-header">
                                <div class="asset__card-value">1.25 ETH</div>
                                <div class="asset__card-fiat">$ 2948.04</div>
                            </div>
                            <div class="asset__card-settings">
                                <button class="button-more">
                                    <span class="button-more__icon"></span>
                                </button>
                           </div>
                            <div class="asset__card-footer">
                                <div class="asset__card-icon">
                                    <img src="./img/cards-icon/ethereum.svg" alt="ethereum">
                                </div>
                                <div class="asset__card-income">+ 0.27%</div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div class="dashboard__market">
                <div class="market">
                    <div class="market__header">
                        <h2 class="title-2">Market is down 0.80%</h2>
                        <div class="market__header-buttons">
                            <button class="dropdown-btn">24h</button>
                            <button class="dropdown-btn">Top gainers</button>
                        </div>
                    </div>
                    <div class="market__table">
                        <table class="market-table" cellpadding="0" cellspacing="0">
                            <thead>
                                <tr>
                                    <th>Name</th>
                                    <th>Price</th>
                                    <th>Chanee</th>
                                    <th class="text-right">Market Cap</th>
                                    <th class="text-center">Watch</th>
                                </tr>
                            </thead>
                            <tbody>
                                <tr>
                                    <td>
                                        <div class="coin">
                                            <div class="coin-icon">
                                                <img src="./img/currency-icons/band.svg" alt="band">
                                            </div>
                                            <div class="coin-text">
                                                <div class="coin-title">Band Protocol</div>
                                                <div class="coin-desc">BAND</div>
                                            </div>
                                        </div>
                                    </td>
                                    <td class="market-table__price">$2.42</td>
                                    <td class="market-table__chanee">+13.38%</td>
                                    <td class="market-table__cap text-right">$399.8M</td>
                                    <td class="market-table__rate text-center"><button><img src="./img/star.svg" alt="Rate"></button></td>
                                </tr>
                                <tr>
                                    <td>
                                        <div class="coin">
                                            <div class="coin-icon">
                                                <img src="./img/currency-icons/vet.svg" alt="vet">
                                            </div>
                                            <div class="coin-text">
                                                <div class="coin-title">VeChain</div>
                                                <div class="coin-desc">VET</div>
                                            </div>
                                        </div>
                                    </td>
                                    <td class="market-table__price">$7.48</td>
                                    <td class="market-table__chanee">+11.19%</td>
                                    <td class="market-table__cap text-right">$152.5M</td>
                                    <td class="market-table__rate text-center"><button><img src="./img/star.svg" alt="Rate"></button></td>
                                </tr>
                                <tr>
                                    <td>
                                        <div class="coin">
                                            <div class="coin-icon">
                                                <img src="./img/currency-icons/aave.svg" alt="aave">
                                            </div>
                                            <div class="coin-text">
                                                <div class="coin-title">Aave</div>
                                                <div class="coin-desc">AAVE</div>
                                            </div>
                                        </div>
                                    </td>
                                    <td class="market-table__price">$0.0184</td>
                                    <td class="market-table__chanee">+7.57%</td>
                                    <td class="market-table__cap text-right">$1.2B</td>
                                    <td class="market-table__rate text-center"><button><img src="./img/star.svg" alt="Rate"></button></td>
                                </tr>
                                <tr>
                                    <td>
                                        <div class="coin">
                                            <div class="coin-icon">
                                                <img src="./img/currency-icons/waves.svg" alt="waves">
                                            </div>
                                            <div class="coin-text">
                                                <div class="coin-title">Waves</div>
                                                <div class="coin-desc">WAVES</div>
                                            </div>
                                        </div>
                                    </td>
                                    <td class="market-table__price">$30.68</td>
                                    <td class="market-table__chanee">+6.80%</td>
                                    <td class="market-table__cap text-right">$399.8M</td>
                                    <td class="market-table__rate text-center"><button><img src="./img/star.svg" alt="Rate"></button></td>
                                </tr>
                            </tbody>
                        </table>
                    </div>
                </div>
                <div class="earn-free-wrapper">
                    <div class="earn-free">
                        <div class="earn-free__title">
                            Earn <span class="earn-free__title-free">free</span> crypto
                            with Coinview Earn!</div>
                        <div class="earn-free__text">Learn about different cryptocurrencies and
                            earn them for free!</div>
                        <div class="earn-free__button-wrapper">
                            <a href="#" class="earn-free__button">Earn now</a>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</body>
</html>