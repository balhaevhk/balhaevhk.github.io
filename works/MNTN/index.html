<!DOCTYPE html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>MNTN</title>
    <link rel="stylesheet" href="main.css">
    <link rel="stylesheet" href="media.css">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500;600&family=Roboto:wght@500;700;900&display=swap" rel="stylesheet">
</head>
<body>
    <header class="header">
        <nav class="nav-wrapper">
            <div class="container-wide">
                <div class="row space-between">
                    <div class="logo">
                        MNTN
                    </div>
                    <div class="nav">
                        <ul>
                            <li><a href="#">Equipment</a></li>
                            <li><a href="#">About us</a></li>
                            <li><a href="#">Blog</a></li>
                        </ul>
                    </div>
                    <a href="#" class="account">
                        <svg class="account-logo" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path fill-rule="evenodd" clip-rule="evenodd" d="M12 2C6.48 2 2 6.48 2 12C2 17.52 6.48 22 12 22C17.52 22 22 17.52 22 12C22 6.48 17.52 2 12 2ZM7.07 18.28C7.5 17.38 10.12 16.5 12 16.5C13.88 16.5 16.51 17.38 16.93 18.28C15.57 19.36 13.86 20 12 20C10.14 20 8.43 19.36 7.07 18.28ZM18.36 16.83C16.93 15.09 13.46 14.5 12 14.5C10.54 14.5 7.07 15.09 5.64 16.83C4.62 15.49 4 13.82 4 12C4 7.59 7.59 4 12 4C16.41 4 20 7.59 20 12C20 13.82 19.38 15.49 18.36 16.83ZM12 6C10.06 6 8.5 7.56 8.5 9.5C8.5 11.44 10.06 13 12 13C13.94 13 15.5 11.44 15.5 9.5C15.5 7.56 13.94 6 12 6ZM12 11C11.17 11 10.5 10.33 10.5 9.5C10.5 8.67 11.17 8 12 8C12.83 8 13.5 8.67 13.5 9.5C13.5 10.33 12.83 11 12 11Z" fill="white"/>
                            <mask id="mask0_1_50" style="mask-type:luminance" maskUnits="userSpaceOnUse" x="2" y="2" width="20" height="20">
                            <path fill-rule="evenodd" clip-rule="evenodd" d="M12 2C6.48 2 2 6.48 2 12C2 17.52 6.48 22 12 22C17.52 22 22 17.52 22 12C22 6.48 17.52 2 12 2ZM7.07 18.28C7.5 17.38 10.12 16.5 12 16.5C13.88 16.5 16.51 17.38 16.93 18.28C15.57 19.36 13.86 20 12 20C10.14 20 8.43 19.36 7.07 18.28ZM18.36 16.83C16.93 15.09 13.46 14.5 12 14.5C10.54 14.5 7.07 15.09 5.64 16.83C4.62 15.49 4 13.82 4 12C4 7.59 7.59 4 12 4C16.41 4 20 7.59 20 12C20 13.82 19.38 15.49 18.36 16.83ZM12 6C10.06 6 8.5 7.56 8.5 9.5C8.5 11.44 10.06 13 12 13C13.94 13 15.5 11.44 15.5 9.5C15.5 7.56 13.94 6 12 6ZM12 11C11.17 11 10.5 10.33 10.5 9.5C10.5 8.67 11.17 8 12 8C12.83 8 13.5 8.67 13.5 9.5C13.5 10.33 12.83 11 12 11Z" fill="white"/>
                            </mask>
                            <g mask="url(#mask0_1_50)">
                            </g>
                            </svg>  
                        Account</a>
                        <button class="nav-button">
                            <span class="nav-button-icon"></span>
                        </button>
                        <div class="mobile-nav">
                            <div class="nav">
                                <ul>
                                    <li><a href="#">Equipment</a></li>
                                    <li><a href="#">About us</a></li>
                                    <li><a href="#">Blog</a></li>
                                </ul>
                            </div>
                            <a href="#" class="account">
                                <svg class="account-logo" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
                                    <path fill-rule="evenodd" clip-rule="evenodd" d="M12 2C6.48 2 2 6.48 2 12C2 17.52 6.48 22 12 22C17.52 22 22 17.52 22 12C22 6.48 17.52 2 12 2ZM7.07 18.28C7.5 17.38 10.12 16.5 12 16.5C13.88 16.5 16.51 17.38 16.93 18.28C15.57 19.36 13.86 20 12 20C10.14 20 8.43 19.36 7.07 18.28ZM18.36 16.83C16.93 15.09 13.46 14.5 12 14.5C10.54 14.5 7.07 15.09 5.64 16.83C4.62 15.49 4 13.82 4 12C4 7.59 7.59 4 12 4C16.41 4 20 7.59 20 12C20 13.82 19.38 15.49 18.36 16.83ZM12 6C10.06 6 8.5 7.56 8.5 9.5C8.5 11.44 10.06 13 12 13C13.94 13 15.5 11.44 15.5 9.5C15.5 7.56 13.94 6 12 6ZM12 11C11.17 11 10.5 10.33 10.5 9.5C10.5 8.67 11.17 8 12 8C12.83 8 13.5 8.67 13.5 9.5C13.5 10.33 12.83 11 12 11Z" fill="white"/>
                                    <mask id="mask0_1_50" style="mask-type:luminance" maskUnits="userSpaceOnUse" x="2" y="2" width="20" height="20">
                                    <path fill-rule="evenodd" clip-rule="evenodd" d="M12 2C6.48 2 2 6.48 2 12C2 17.52 6.48 22 12 22C17.52 22 22 17.52 22 12C22 6.48 17.52 2 12 2ZM7.07 18.28C7.5 17.38 10.12 16.5 12 16.5C13.88 16.5 16.51 17.38 16.93 18.28C15.57 19.36 13.86 20 12 20C10.14 20 8.43 19.36 7.07 18.28ZM18.36 16.83C16.93 15.09 13.46 14.5 12 14.5C10.54 14.5 7.07 15.09 5.64 16.83C4.62 15.49 4 13.82 4 12C4 7.59 7.59 4 12 4C16.41 4 20 7.59 20 12C20 13.82 19.38 15.49 18.36 16.83ZM12 6C10.06 6 8.5 7.56 8.5 9.5C8.5 11.44 10.06 13 12 13C13.94 13 15.5 11.44 15.5 9.5C15.5 7.56 13.94 6 12 6ZM12 11C11.17 11 10.5 10.33 10.5 9.5C10.5 8.67 11.17 8 12 8C12.83 8 13.5 8.67 13.5 9.5C13.5 10.33 12.83 11 12 11Z" fill="white"/>
                                    </mask>
                                    <g mask="url(#mask0_1_50)">
                                    </g>
                                    </svg>  
                                Account</a>
                        </div>
                </div>
            </div>
        </nav>
        <a href="#" class="hero">
            <div class="hero-category">A Hiking guide</div>
            <h1 class="hero-title">Be prepared for the Mountains and beyond!</h1>
            <p class="hero-scroll">scroll down</p>
        </a>
    </header>
    <main>
        <div class="container">
            <article class="article" >
                <div class="article-text" data-number="01">
                    <div class="article-subtitle">Get Started</div>
                    <div class="article-title">What level of  hiker are you?</div>
                    <p>Determining what level of hiker you are can be an important tool when planning future hikes. This hiking level guide will help you plan hikes according to different hike ratings set by various websites like All Trails and Modern Hiker. What type of hiker are you – novice, moderate, advanced moderate, expert, or expert backpacker?</p>
                    <a href="#" class="article-read-more">read more</a>
                </div>
                <div class="article-img"><img src="./img/01.jpg" alt="01"></div>
            </article>
            <article class="article" >
                <div class="article-text" data-number="02">
                    <div class="article-subtitle">Hiking Essentials</div>
                    <div class="article-title">Picking the right Hiking Gear!</div>
                    <p>The nice thing about beginning hiking is that you don’t really need any special gear, you can probably get away with things you already have. Let’s start with clothing. A typical mistake hiking beginners make is wearing jeans and regular clothes, which will get heavy and chafe wif they get sweaty or wet.</p>
                    <a href="#" class="article-read-more">read more</a>
                </div>
                <div class="article-img"><img src="./img/02.jpg" alt="02"></div>
            </article>
            <article class="article" >
                <div class="article-text" data-number="03">
                    <div class="article-subtitle">where you go is the key</div>
                    <div class="article-title">Understand Your Map & Timing</div>
                    <p>To start, print out the hiking guide and map. If it’s raining, throw them in a Zip-Lock bag. Read over the guide, study the map, and have a good idea of what to expect. I like to know what my next landmark is as I hike. For example, I’ll read the guide and know that say, in a mile, I make a right turn at the junction..</p>
                    <a href="#" class="article-read-more">read more</a>
                </div>
                <div class="article-img"><img src="./img/03.jpg" alt="03"></div>
            </article>
        </div>
    </main>
    <footer class="footer">
        <div class="container">
            <div class="footer-row">
                <div class="footer-copyright">
                    <div class="footer-copyright-logo">
                        <div class="logo">MNTN</div>
                    </div>
                    <p class="footer-copyright-desc">Get out there & discover your next slope, mountain & destination!</p>
                    <p class="footer-copyright-terms">Copyright 2019 MNTN, Inc. Terms & Privacy</p>
                </div>
                <div class="footer-nav-wrapper">
                    <div class="footer-nav">
                        <h4 class="footer-nav-title">More on The Blog</h4>
                        <ul>
                            <li><a href="#">About MNTN</a></li>
                            <li><a href="#">Contributors & Writers</a></li>
                            <li><a href="#">Write For Us</a></li>
                            <li><a href="#">Contact Us</a></li>
                            <li><a href="#">Privacy Policy</a></li>
                        </ul>
                    </div>
                    <div class="footer-nav">
                        <h4 class="footer-nav-title">More on MNTN</h4>
                        <ul>
                            <li><a href="#">The Team</a></li>
                            <li><a href="#">Jobs</a></li>
                            <li><a href="#">Press</a></li>
                        </ul>
                    </div>
                </div>
            </div>
        </div>
    </footer>
    <script src="./main.js"></script>
</body>
</html>