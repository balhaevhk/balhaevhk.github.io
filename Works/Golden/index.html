<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="CSS/style.css">
    <title>Верстка HTML + CSS За 2 Часа. Из PSD. С Нуля. Полная Запись.</title>
</head>
<body>
    <header>
        <div class="container">
            <div class="heading clearfix">
                <img src="img/logo.png" alt="Логотип" class="logo">;
                <nav class="menu">
                    <ul>
                        <li><a href="#">Home</a></li>
                        <li><a href="#">Services</a></li>
                        <li><a href="#">Portfolio</a></li>
                        <li><a href="#">About</a></li>
                        <li><a href="#">Contact</a></li>
                    </ul>
                </nav>
            </div>
            <div class="titles">
                <div class="titles_first">Welcome To Our Studio</div>
                <h1>It's to meet you</h1>
            </div>
            <a href="#" class="button">Tell me more</a>
            
        </div>
    </header>
    <section>
        <div class="container">
            <div class="title">
                <h2>Services</h2>
                <p>Lorem ipsum dolor sit amet consectetur.</p>
            </div>
            <div class="services clearfix">
                <div class="services_item">
                    <img src="img/icon1.png" alt="Услуга">
                    <h3>E-Commerce</h3>
                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique soluta modi cupiditate minus quidem, possimus eum magnam repudiandae.</p>
                </div>
                <div class="services_item">
                    <img src="img/icon2.png" alt="Услуга">
                    <h3>Responsive-Web</h3>
                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique soluta modi cupiditate minus quidem, possimus eum magnam repudiandae.</p>
                </div><div class="services_item">
                    <img src="img/icon3.png" alt="Услуга">
                    <h3>Web Security</h3>
                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique soluta modi cupiditate minus quidem, possimus eum magnam repudiandae.</p>
                </div>
            </div>
        </div>

    </section>
    <section class="portfolio">
        <div class="container">
            <div class="title">
                <h2>Our portfolio</h2>
                <p>Lorem ipsum dolor sit amet consectetur.</p>
            </div>
            <div class="works clearfix">
                <img src="img/img1.jpg" alt="Работа">
                <img src="img/img2.jpg" alt="Работа">
                <img src="img/img3.jpg" alt="Работа">
            </div>
        </div>
    </section>
    <section>
        <div class="container">
            <div class="title">
                <h2>About us</h2>
                <p>Lorem ipsum dolor sit amet consectetur.</p>
            </div>
            <div class="way">
                <div class="step_left">
                    <img src="img/about1.jpg" alt="Путь">
                    <div class="text">
                        <h3><span class="time">jule 2010</span><br>our humble beginnings</h3>
                        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Iure repellendus, doloribus voluptatem incidunt voluptatibus neque dolores perspiciatis tempora, debitis quo fugiat deserunt nulla tenetur totam, doloremque vel iste corrupti.</p>
                    </div>
                </div>
                <div class="step_right">
                    <img src="img/about2.jpg" alt="Путь">
                    <div class="text">
                        <h3><span class="time">january 2011</span><br>facing startup battles</h3>
                        <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptatibus excepturi accusantium laborum esse corporis possimus et. Explicabo esse ratione similique voluptate pariatur quos a ipsam, animi rem ut possimus illo eos aut magnam neque odit quo repellat! Labore, mollitia quos. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptatibus excepturi accusantium laborum esse corporis possimus et. Explicabo esse ratione similique voluptate pariatur quos a ipsam, animi rem ut possimus illo eos aut magnam neque odit quo repellat! Labore, mollitia quos.</p>
                    </div>
                </div>
                <div class="step_left">
                    <img src="img/about3.jpg" alt="Путь">
                    <div class="text">
                        <h3><span class="time">december 2012</span><br>enter the dark days</h3>
                        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Excepturi atque sint velit assumenda fuga et ipsum consectetur eligendi nesciunt veritatis. Dolorem, minus nihil. Alias, id.</p>
                    </div>
                </div>
                <div class="step_right">
                    <img src="img/about4.jpg" alt="Путь">
                    <div class="text">
                        <h3><span class="time">february 2014</span><br>our triumph</h3>
                        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Cupiditate non, nisi molestiae perferendis et quis error reprehenderit ipsum nam ipsam recusandae unde ullam voluptatum ab, facere itaque! Totam eos odio eveniet?</p>
                    </div>
                </div>
            </div>
        </div>

    </section>
    <footer>
        <div class="container">
            <div class="title">
                <p>&copy; Copyright 2014 FreebiesXpress.com<br>The layout of the site was made by Hasan Balhaev</p>
                <p></p>
            </div>
        </div>
    </footer>
</body>
</html>